rpmbuild/bin/copr-builder

```diff
@@ -0,0 +1,222 @@
+#! /usr/bin/python3
+
+"""
+Allow users to control their Copr builder instance
+https://github.com/fedora-copr/debate/tree/main/user-ssh-builders
+"""
+
+import argparse
+from datetime import datetime, timedelta
+from contextlib import suppress
+from copr_common.helpers import timedelta_to_dhms
+
+
+RPMBUILD_PID_PATH = "/var/lib/copr-rpmbuild/pid"
+EXPIRATION_PATH = "/run/copr-builder-expiration"
+
+# See also background_worker_build.py
+DEFAULT_EXPIRATION = 60 * 60
+MAX_EXPIRATION = 60 * 60 * 24 * 2
+
+
+def cmd_help():
+    """
+    Print full instructions for working with the builder instance.
+    Ideally we would have this in MOTD but that would cause problems in cases
+    like `if $(ssh stroj cat foo|head -n1) == "foo"`. So instead, we instruct
+    users to run `copr-builder help` manually.
+    """
+    # TODO Should this be in the documentation instead?
```

Comment:
Not sure if "instead", but maybe "also"? :-)

Reply:
I mentioned the `copr-builder help` command in the Copr web UI, let's start from there. Maybe we will move it to the documentation page in the future.